src/openMVG/exif/sensor_width_database/sensor_width_camera_database.txt

```diff
@@ -15,6 +15,7 @@ Acer CS-5531;5.75
 Acer CS-6530;5.75
 Acer CS-6531;5.75
 Acer CU-6530;5.75
+samsung SM-G973F;5.75
```

Comment:
Could we remove this modification from this PR?

Reply:
I'm sorry, this was included due to an oversight. Fixed now.